### DISCUSSION TITLE
Discussions in recent activity

### DISCUSSION THREAD

**Discussion by iMrDJAi** [lowlighter/metrics#1256] [General]
Discussions and comments on discussions aren't displayed in the **Recent Activity** section. Is there a way to achieve that?

**Comment by lowlighter** [comment]
Hi 👋 !

It's not supported yet and it would be nice to have them displayed, but discussions events aren't listed in the [list of github event](https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types) so it might not be possible to implement it yet unfortunately

It might be merged with issues event though, since internally github considers everything as issues (like https://github.com/lowlighter/metrics/issues/1256 redirect here)

Seems there's some event that have been added
- PullRequestReviewThreadEvent
- SponsorshipEvent

So it might be the good time to check if discussions event can be fecthed

**Reply by iMrDJAi** [reply]
Thank you so much for clarifying that!
